common/ratelimit/limiter.go

```diff
@@ -109,7 +119,14 @@ func (d *rateLimiter) checkAllowed(ctx context.Context, params common.RequestPar
 		bucketParams.BucketLevels[i] = getBucketLevel(bucketParams.BucketLevels[i], size, interval, deduction)
 		allowed = allowed && bucketParams.BucketLevels[i] > 0
 
-		d.logger.Debug("Bucket level", "key", params.RequesterID, "prevLevel", prevLevel, "level", bucketParams.BucketLevels[i], "size", size, "interval", interval, "deduction", deduction, "allowed", allowed)
+		// Check if the user is authenticated before adding to metrics
```

Comment:
I think it would be useful to also capture the system-level ratelimits. 

I guess this could be done by adding another field to the params struct indicating if it is a system or account level rate limit. Alternatively, we could replace `IsAuthenticated` with a different semantic, like `ShouldRecordMetrics`.

Reply:
Added system level ratelimits + parsed the string to get more labels.
```
# HELP rate_limiter_bucket_levels Current level of each bucket for rate limiting
# TYPE rate_limiter_bucket_levels gauge
rate_limiter_bucket_levels{account_key="18.208.69.153",account_type="ip",bucket_index="0",quorum="0",type="account_blob_rate"} 6e+11
rate_limiter_bucket_levels{account_key="18.208.69.153",account_type="ip",bucket_index="0",quorum="0",type="account_throughput"} 6e+11
rate_limiter_bucket_levels{account_key="18.208.69.153",account_type="ip",bucket_index="0",quorum="1",type="account_blob_rate"} 6e+11
rate_limiter_bucket_levels{account_key="18.208.69.153",account_type="ip",bucket_index="0",quorum="1",type="account_throughput"} 6e+11
# HELP rate_limiter_system_bucket_levels Current level of each bucket for system rate limiting
# TYPE rate_limiter_system_bucket_levels gauge
rate_limiter_system_bucket_levels{bucket_index="0",quorum="0",type="system_blob_rate"} 6e+11
rate_limiter_system_bucket_levels{bucket_index="0",quorum="0",type="system_throughput"} 6e+11
rate_limiter_system_bucket_levels{bucket_index="0",quorum="1",type="system_blob_rate"} 6e+11
rate_limiter_system_bucket_levels{bucket_index="0",quorum="1",type="system_throughput"} 6e+11
```